### PR TITLE
Updating working groups links to community wiki

### PR DESCRIPTION
**Pull request by caiohsramos**
##### SUMMARY
Updating working groups links (on https://docs.ansible.com/ansible/devel/community/communication.html) to community wiki and adding Working Groups wiki link (https://github.com/ansible/community/wiki#working-groups)

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
communication.rst

##### ADDITIONAL INFORMATION
Most of Working Group links were broken with invalid URLs

+label: docsite_pr

### CODE REVIEW

File: docs/docsite/rst/community/communication.rst [hunk 1]
```diff
@@ -47,27 +47,27 @@ General channels
...
+- `Amazon (AWS) Working Group <https://github.com/ansible/community/wiki/AWS>`_ - ``#ansible-aws``
+- `Ansible Lockdown Working Group <https://github.com/ansible/community/wiki/Lockdown>`_ | `gh/ansible/ansible-lockdown <https://github.com/ansible/ansible-lockdown>`_ - ``#ansible-lockdown``- Security playbooks/roles
 - `AWX Working Group <https://github.com/ansible/awx>`_ - ``#ansible-awx`` - Upstream Ansible Tower
-- `Azure Working Group <https://github.com/ansible/community/tree/master/group-azure>`_ - ``#ansible-azure``
-- `Community Working Group <https://github.com/ansible/community/tree/master/group-community>`_ - ``#ansible-community`` - Including Meetups
-- `Container Working Group <https://github.com/ansible/community/tree/master/group-container>`_ - ``#ansible-container``
-- `Contributor Experience Working Group <https://github.com/ansible/community/tree/master/group-contributor-experience>`_ - ``#ansible-community``
-- `Docker Working Group <https://github.com/ansible/community/issues/408>`_ - ``#ansible-devel``
```
Comment (jamescassell) [inline docs/docsite/rst/community/communication.rst:59]: This link is still active.

Comment (caiohsramos) [reply docs/docsite/rst/community/communication.rst:59]: The new link for the Docker Working Group (https://github.com/ansible/community/wiki/Docker) has already a reference for https://github.com/ansible/community/issues/408 at "Docker meeting agenda".

Comment (gundalow) [reply docs/docsite/rst/community/communication.rst:59]: When I originally wrote this we didn't have a wiki page. Thanks for updating this.

**Review by gundalow — APPROVED** [review]
Thank you for updating this.

**Review by jamescassell — APPROVED** [review]

**Comment by acozine** [comment]
Thanks @caiohsramos for helping to update things!

**Pull request merged**